test/e2e-serving-upgrade-tests.sh

```diff
@@ -69,8 +66,17 @@ function test_setup() {
   cd ${OPERATOR_DIR}
 }
 
+# Create test namespaces for serving
+function create_test_namespace_serving() {
+  kubectl get ns ${SERVING_TESTS_NAMESPACE} || kubectl create namespace ${SERVING_TESTS_NAMESPACE}
+  kubectl get ns ${SERVING_TESTS_ALT_NAMESPACE} || kubectl create namespace ${SERVING_TESTS_ALT_NAMESPACE}
+}
+
 # Skip installing istio as an add-on.
-initialize "$@" --skip-istio-addon
+# Temporarily increasing the cluster size for serving tests to rule out
+# resource/eviction as causes of flakiness.
+# Pin to 1.18 since scale test is super flakey on 1.19
```

Comment:
Do we even run the scale tests here?

Reply:
We ran the tests including:
```
test/upgrade: TestServingUpgrades/PreUpgradeTests/ServicePreUpgradeAndScaleToZeroTest 
test/upgrade: TestServingUpgrades/PreUpgradeTests/InitialScalePreUpgradeTest
test/upgrade: TestServingUpgrades/PostUpgradeTests/PostUpgradeFromScaleToZeroTest
test/upgrade: TestServingUpgrades/PostUpgradeTests/InitialScalePostUpgradeTest
```
They are all indicated with `scale`.

Reply:
Those are not the scale tests that are causing trouble in Serving though. They'd be in the scale package, which I believe we're not running here.

Reply:
then, it is not running the scale tests.